src/data/nat/parity.lean

```diff
@@ -89,6 +89,12 @@ end
 theorem even.add {m n : ℕ} (hm : even m) (hn : even n) : even (m + n) :=
 even_add.2 $ by simp only [*]
 
+theorem even_add' {m n : ℕ} : even (m + n) ↔ (odd m ↔ odd n) :=
+by rw [even_add, even_iff_not_odd, even_iff_not_odd, not_iff_not]
+
+theorem even.add' {m n : ℕ} (hm : odd m) (hn : odd n) : even (m + n) :=
```

Comment:
```suggestion
theorem odd.add_odd {m n : ℕ} (hm : odd m) (hn : odd n) : even (m + n) :=
```
using `odd.*` makes it possible to use dot-notation.

Reply:
Thanks for letting me know! I'll make these changes.